src/developer/web-api/tracker.md

```diff
@@ -611,32 +632,191 @@ Examples for the **FLAT** and the **NESTED** versions of the payload are listed
+
+#### Delete attribute values
+
+The following deletes one of the attribute values of a [tracked entity](#payload):
+
+    POST /api/tracker?async=false
+
+```json
+{
+  "trackedEntities": [
+    {
+      "trackedEntity": "PQfMcpmXeFE",
+      "trackedEntityType": "nEenWmSyUEp",
+      "orgUnit": "DiszpKrYNg8",
+      "attributes": [
+        {
+          "attribute": "w75KJ2mc4zz",
+          "value": null
+        }
+      ]
+    }
+  ]
+}
+```
+
+#### Update data values
+
+The following updates one of the data values of an [event](#payload):
+
+    POST /api/tracker?async=false
+
+```json
+{
+  "events": [
+    {
+      "event": "ZwwuwNp6gVd",
+      "dataValues": [
+        {
+          "dataElement": "bx6fsa0t90x",
+          "value": "true"
+        }
+      ],
+      "attributeOptionCombo": "HllvX50cXC0",
+      "attributeCategoryOptions": "xYerKDKCefk",
+      "enrollment": "MNWZ6hnuhSw",
+      "enrollmentStatus": "ACTIVE",
+      "occurredAt": "2019-08-01T00:00:00.000",
+      "orgUnit": "y77LiPqLMoq",
+      "program": "IpHINAT79UW",
+      "programStage": "A03MvHHogjR",
+      "scheduledAt": "2019-08-19T13:59:13.688",
+      "status": "ACTIVE",
+      "trackedEntity": "Kj6vYde4LHh"
+    }
+  ]
 }
 ```
 
+#### Delete data values
+
+The following deletes one of the data values of an [event](#payload):
+
+    POST /api/tracker?async=false
+
+```json
+{
+  "events": [
+    {
+      "event": "ZwwuwNp6gVd",
+      "dataValues": [
+        {
+          "dataElement": "bx6fsa0t90x",
+          "value": null
+        }
+      ],
+      "attributeOptionCombo": "HllvX50cXC0",
+      "attributeCategoryOptions": "xYerKDKCefk",
+      "enrollment": "MNWZ6hnuhSw",
+      "enrollmentStatus": "ACTIVE",
+      "occurredAt": "2019-08-01T00:00:00.000",
+      "orgUnit": "y77LiPqLMoq",
+      "program": "IpHINAT79UW",
+      "programStage": "A03MvHHogjR",
+      "scheduledAt": "2019-08-19T13:59:13.688",
+      "status": "ACTIVE",
+      "trackedEntity": "Kj6vYde4LHh"
+    }
+  ]
+}
+```
+
+### Delete
+
+Make a `POST` to `/api/tracker` with `importStrategy` set to `DELETE`. The payload should include
+only the UIDs of the `trackedEntities`, `enrollments`, `events` or `relationships` you wish to
+delete.
+
+The following deletes the events created with [this payload](#payload):
+
+    POST /api/tracker?async=false&importStrategy=delete
+
+```json
+{
+  "events": [
+    {
+      "event": "ZwwuwNp6gVd",
+    },
+    {
+      "event": "XwwuwNp6gVE",
+    }
+  ]
+}
+```
+
+The following deletes the tracked entities and all its child tracker objects which are enrollments,
+events and relationships:
+
+    POST /api/tracker?async=false&importStrategy=delete
+
+```json
+{
+  "trackedEntities": [
+    {
+      "trackedEntity": "Kj6vYde4LHh",
+    },
+    {
+      "trackedEntity": "Gjaiu3ea38E",
+    }
+  ]
+}
+```
+
+All the children of a tracker object will be deleted if the user making the request has the
+`DELETE_CASCADE` authority.
```

Comment:
The 2 authorities needed here are `F_TEI_CASCADE_DELETE` and `F_ENROLLMENT_CASCADE_DELETE`